modules/store/src/connextStore.ts

```diff
@@ -83,11 +81,10 @@ export class ConnextStore implements IClientStore {
       case StoreTypes.File: {
         this.internalStore = new KeyValueStorage(
           new WrappedSequelizeStorage(
+            `sqlite:${opts.fileDir}/${storeDefaults.SQLITE_STORE_NAME}`,
```

Comment:
Shouldn't we just remove StoreTypes.File all together? I'm pretty sure only rest-api-client was using it

Reply:
`File` is basically an alias for the SQLite store. We can change the name to be more specific.